spec/controllers/admin/order_cycles_controller_spec.rb

```diff
@@ -137,7 +137,7 @@ module Admin
                 select: {
                   enterprise_fees: 3,
                   enterprise_groups: 1,
-                  enterprises: 22,
+                  enterprises: 24,
```

Comment:
In this test the user manages the coordinating enterprise, right? It feels like the number of enterprises queried here shouldn't have changed in that case? :thinking:

Reply:
I think this is coming from the queries that I added to `managed_participating_hubs` & `managed_participating_producers`

`OrderCyclesController` -> `OrderCycleSerializer` -> `OrderCyclePermissions`

Reply:
I'll revert this commit since this PR will not solve that part of the issue.